content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -591,25 +595,27 @@ appropriate arguments.
 See the [`kubeadm reset`](/docs/reference/setup-tools/kubeadm/kubeadm-reset/)
 reference documentation for more information about this subcommand and its
 options.
+{{% /capture %}}
 
-## Maintaining a cluster {#lifecycle}
-
-Instructions for maintaining kubeadm clusters (e.g. upgrades,downgrades, etc.) can be found [here.](/docs/tasks/administer-cluster/kubeadm)
-
-## Explore other add-ons {#other-addons}
-
-See the [list of add-ons](/docs/concepts/cluster-administration/addons/) to explore other add-ons,
-including tools for logging, monitoring, network policy, visualization &amp;
-control of your Kubernetes cluster.
+{{% capture discussion %}}
 
 ## What's next {#whats-next}
 
 * Verify that your cluster is running properly with [Sonobuoy](https://github.com/heptio/sonobuoy)
+* <a id="lifecycle" />See [Upgrading kubeadm clusters](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
+  for details about upgrading your cluster using kubeadm.
 * Learn about kubeadm's advanced usage in the [kubeadm reference documentation](/docs/reference/setup-tools/kubeadm/kubeadm)
 * Learn more about Kubernetes [concepts](/docs/concepts/) and [`kubectl`](/docs/user-guide/kubectl-overview/).
-* Configure log rotation. You can use **logrotate** for that. When using Docker, you can specify log rotation options for Docker daemon, for example `--log-driver=json-file --log-opt=max-size=10m --log-opt=max-file=5`. See [Configure and troubleshoot the Docker daemon](https://docs.docker.com/engine/admin/) for more details.
 * See the [Cluster Networking](/docs/concepts/cluster-administration/networking/) page for a bigger list
 of Pod network add-ons.
+* <a id="other-addons" />See the [list of add-ons](/docs/concepts/cluster-administration/addons/) to
+  explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp;
+  control of your Kubernetes cluster.
+* Configure log rotation. You can use **logrotate** for that. When using Docker,
+  you can specify log rotation options for Docker daemon, for example:
+  `--log-driver=json-file --log-opt=max-size=10m --log-opt=max-file=5`.  
+  See [Configure and troubleshoot the Docker daemon](https://docs.docker.com/engine/admin/)
+  for more details.
```

Comment:
i think we can remove these docker logrotate related instructions from the guide:
regular users don't need to do that.

Reply:
Fair enough. There's also CRIContainerLogRotation to account for so the existing text is neither necessary nor sufficient.